code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -221,7 +221,7 @@
 			log_combat(user, src, "emagged")
 		return
 	else //Bot is unlocked, but the maint panel has not been opened with a screwdriver yet.
-		to_chat(user, "<span class='warning'>You need to open maintenance panel first!</span>")
+		to_chat(user, "<span class='warning'>You need to open [src]'s maintenance panel first!</span>")
```

Comment:
use `p_their()` instead to respect Officer Beepsky's preferred pronouns please.

Reply:
Please mark the PR as draft if you request changes 🙂